packages/react/src/AvatarStack/AvatarStack.tsx

```diff
@@ -249,6 +260,11 @@ const AvatarStack = ({
     )
   }
 
+  useEffect(() => {
+    const hasInteractive = hasInteractiveNodes(avatarStackContainer.current, 'div[tabindex]')
+    setHasInteractiveChildren(hasInteractive)
+  }, [])
```

Comment:
With the move to not running on `children` changing, wanted to ask if this is something that should be setup in a mutation observer for when contents change in case the "hasInteractiveNodes" check could change?

Reply:
We could! I was thinking that we'd know whether to make the container interactive or not based on the initial render as I don't think there are cases where interactive items are rendered in after the first render. But we could add a mutation observer just to be safe.